moveit_core/collision_detection/include/moveit/collision_detection/world.h

```diff
@@ -109,7 +111,7 @@ namespace collision_detection
     ObjectConstPtr getObject(const std::string &id) const;
 
     /** iterator over the objects in the world. */
-    typedef std::map<std::string, ObjectPtr>::const_iterator const_iterator;
+    typedef std::map<std::string, ObjectConstPtr>::const_iterator const_iterator;
```

Comment:
Note that this is not really required, since the two types are actually the same. Not sure why I also reverted this to be honest...